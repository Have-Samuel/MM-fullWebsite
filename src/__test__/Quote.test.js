import renderer from 'react-test-renderer';
import Quote from '../pages/Home';

it('render correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
