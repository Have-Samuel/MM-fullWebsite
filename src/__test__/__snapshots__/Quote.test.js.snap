// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="main-content"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Odit, facilis excepturi. Accusantium nulla cupiditate voluptatem sapiente aliquid illo eum repellendus in quia modi laboriosam vel aperiam fugiat, sequi omnis hic.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur harum itaque ipsam perspiciatis, asperiores, magnam ex, qui temporibus dolorum nisi sequi sunt illo. Quidem libero vitae debitis laborum quod! Itaque.
  </p>
</div>
`;
